docs/source/docker/integration/docker_trigger_from_api.rst

```diff
@@ -0,0 +1,184 @@
+
+
+Requirements
+------------
+This recipe requires that you already have a dataset in the Lightly Platform
+configured to use the data in your AWS S3 bucket. Create such a dataset in 2 steps:
+
+1. `Create a new dataset <https://app.lightly.ai/dataset/create>`_ in Lightly.
+   Make sure that you choose the input type `Images` or `Videos` correctly,
+   depending on the type of files in your cloud storage bucket.
+2. Edit your dataset, select the storage source as your datasource and fill out the form.
+   In our example we use an S3 bucket.
+
+    .. figure:: ../../getting_started/resources/LightlyEdit2.png
+        :align: center
+        :alt: Lightly S3 connection config
+        :width: 60%
+
+        Lightly S3 connection config
+
+If you don`t know how to fill out the form, follow the full tutorial to
+`create a Lightly dataset connected to your S3 bucket <https://docs.lightly.ai/getting_started/dataset_creation/dataset_creation_aws_bucket.html>`_.
+
+Furthermore, you should have access to a machine running docker. Ideally, it 
+also has a CUDA-GPU. A fast GPU will speed up the process significantly, 
+especially for large datasets.
+
+
+Download the Lightly Docker
+---------------------------
+Next, the Lightly Docker should be installed. Please follow the instructions for
+the :ref:`ref-docker-setup`.
+
+
+Register the Lightly Docker as a Worker
+---------------------------------------
+To control the Lightly Docker from the API you have to register it as a worker.
+We can simply go to the Lightly web app and click on My Docker Runs --> My Compute Workers.
+Or just click on the direct link here: `Docker Workers <https://app.lightly.ai/docker/workers>`__
+
+.. image:: ../getting_started/images/docker_workers_overview_empty.png
+
+Click on "Register" in the bottom right corner and enter a name for your worker.
+After confirmation the worker should show up in the worker list.
+
+.. image:: ../getting_started/images/docker_workers_overview_registered.png
+
+Copy the *worker id* and head over to your terminal. You can now start the docker
+with the worker id and it will connect to the API and wait for jobs. To start
+the docker execute the following command: 
+
+.. code-block:: console
+
+    docker run --gpus all --rm -it \
+        -v ${OUTPUT_DIR}:/home/output_dir \
+        lightly/sampling:latest \
+        token=${YOUR_LIGHTLY_PLATFORM_TOKEN} \
+        worker_id=${YOUR_WORKER_ID}
+
+The state of the worker on the `Docker Workers <https://app.lightly.ai/docker/workers>`__
+page should now indicate that the worker is in an idle state.
+
+
+Triggering a Job through the API
+--------------------------------
+
+To trigger a new job you can click on the schedule run button on the dataset
+overview as shown in the screenshot below:
+
+.. image:: images/schedule-compute-run.png
+
+After clicking on the button you will see a wizzard to configure the the parameters
+for the job.
+
+.. image:: images/schedule-compute-run-config.png
```

Comment:
You could change the image to the updated one:
https://user-images.githubusercontent.com/20324507/156338271-ea4ee2a7-b3d7-48da-9d3a-76f87c3c0e97.jpeg